src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/NetworkManager.cs

```diff
@@ -174,7 +165,33 @@ protected void OnReceive(IncomingMessage data)
             {
                 string command = reader.ReadString();
 
-                NotifyCommand(data.Endpoint, command, reader, data.Size - (int)stream.Position);
+                NotifyCommand(data.Connection, command, reader, data.Size - (int)stream.Position);
+            }
+        }
+
+        private void CreateConnectionManager()
+        {
+            if (connectionManagerGameObject == null)
+            {
+                var prefab = defaultConnectionManagerPrefab;
+                if (NetworkConfigurationSettings.IsInitialized &&
+                    NetworkConfigurationSettings.Instance.OverrideConnectionManagerPrefab != null)
+                {
+                    prefab = NetworkConfigurationSettings.Instance.OverrideConnectionManagerPrefab;
+                }
+
+                if (prefab == null)
+                {
+                    throw new MissingComponentException("Network connection manager prefab wasn't specified. DeviceInfoBroadcaster will not work correctly.");

```

Comment:
>DeviceInfoBroadcaster [](start = 109, length = 21)

DeviceInfoBroadcaster looks like it might have been a copy-pasted type name from somewhere else since this is on NetworkManager